src/main/webapp/index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <script src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script src="script.js"></script>
+    <script src="contextServlet.js"></script>
```

Comment:
I think the script.js and contextServlet.js should both be loaded?

Reply:
How are these two script get combined in front end then?

Reply:
Added both to HTML - I'm pretty sure each js file is compiled into one large file when run. Splitting up the files is more just for organization and clarity

Reply:
sg!!